config/production.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
```

Comment:
I can change it for staging/production but at this point the tutorial explicitly mentions to change dev/test config files:
>We don't plan on deploying anything yet so the staging and production config variables do not need to be defined right now.